src/askjsx/lib/For.tsx

```diff
@@ -4,7 +4,7 @@ askjsx;
 
 export function For({
   initialization,
-  condition,
+  condition = true,
```

Comment:
@czerwinskilukasz1 
wow First time see on object assignment `=` its correct or should be `:` ?

Reply:
ES2015